modules/material-management/src/types.ts

```diff
@@ -156,3 +156,7 @@ export interface AwsEsdkAsymmetricKeyDetails {
    */
   namedCurve?: string
 }
+
+export interface ErrorPlus {
+  errPlus?: Error | string | any
+}
```

Comment:
I don't understand this name

Reply:
ErrorPlus, as in Error or More. 

Alternative: Catchable? (as in something that can be caught; like a pokemon.)